app/modules/kit-library/assets/js/hooks/use-kit-call-to-action.js

```diff
@@ -23,10 +25,8 @@ export default function useKitCallToAction( kitAccessTier ) {
 	// SubscriptionPlan can be null when the context is not filled (can be happened when using back button in the browser.)
 	const subscriptionPlan = useMemo( () => settings.subscription_plans?.[ kitAccessTier ], [ settings, kitAccessTier ] );
 
-	// Free user should see a generic "Pro" badge.
-	const badgeLabel = userAccessTier === TIERS.free
-		? settings.subscription_plans?.essential.label
-		: subscriptionPlan?.label;
+	subscriptionPlan.label = PromotionChipText[ tierKey ];
+	subscriptionPlan.color = TIERS.free === kitAccessTier ? '#467FF7' : '#92003B';
```

Comment:
Maybe have const variables for color codes?

Reply:
I put them in the scss file and added classes, WDYT?